src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -551,6 +551,11 @@
 			<method signature="System.Void .ctor(System.Object)" />
 		</type>
 
+		<!-- Used by metadata update -->
+		<type fullname="System.Runtime.CompilerServices.RuntimeFeature">
+			<method name="LoadMetadataUpdate" />
+		</type>
+
```

Comment:
Why does this need to be retained ?

Reply:
Because it will only be called from a debugger session.